operation.go

```diff
@@ -0,0 +1,6 @@
+package merkletree
+
+type Operation struct {
```

Comment:
> Since the CONIKS server will be handling registration and key change operations, and it will have to order these, it may make more sense to define Operation at the server level only?

Yes. I already fixed this in my latest commit.